ocp_resources/resource.py

```diff
@@ -498,8 +515,10 @@ def _prepare_resources(cls, dyn_client, singular_name, *args, **kwargs):
 
         get_kwargs = {"singular_name": singular_name} if singular_name else {}
         return dyn_client.resources.get(
-            kind=cls.kind, api_version=cls.api_version, **get_kwargs
-        ).get(*args, **kwargs)
+            kind=cls.kind,
+            api_version=cls.api_version,
+            **get_kwargs,
+        ).get(*args, **kwargs, timeout_seconds=60)
```

Comment:
```suggestion
        ).get(*args, **kwargs, timeout_seconds=self.timeout_seconds)
```

Reply:
:)

Reply:
I took the suggestions too fast. _prepare_resources() is a class method so this won't work. and I can't use cls. timeout_seconds here.

Reply:
@dbasunag `cls.` should work, you need to define it outside `__init__`

Reply:
Ok, added it as a class attribute with a default value.